### DISCUSSION TITLE
How to use SingleFile on Android?

### DISCUSSION THREAD

**Discussion by gildas-lormeau** [gildas-lormeau/SingleFile#650] [Ideas]
You can actually install SingleFile on 
- [Firefox for Android Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) by following this procedure: https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/
- [Yandex Browser](https://play.google.com/store/apps/details?id=com.yandex.browser) by installing SingleFile from the store of Chrome
- [Kiwi Browser](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser) by installing SingleFile from the store of Chrome

**Comment by freemedom** [comment]
Here is an app that implements part of this feature. It mainly focuses on capturing text from native apps, not just html, through the accessibility API. Currently, it only supports saving, it can't restore the original layout or appearance.
https://github.com/freemedom/TextGrabber
